buildSrc/subprojects/jvm/src/main/kotlin/gradlebuild/unittest-and-compile.gradle.kts

```diff
@@ -208,7 +208,7 @@ fun configureTests() {
 
         val testName = name
 
-        if (BuildEnvironment.isCiServer && !this.javaClass.simpleName.endsWith("PerformanceTest")) {
+        if (BuildEnvironment.isCiServer) {
             retry {
                 maxRetries.set(1)
```

Comment:
```suggestion
                maxRetries.convention(1)
```

I think there is an order problem here. Which configuration takes place first? Wouldn't it be better to use a convention here, so anybody which sets it then surely overrides the value?

Reply:
Good to know!